packages/affiliates/apps/DappMiningRewards.js

```diff
@@ -18,15 +20,12 @@ async function App(config) {
   const result = await dappmining.getRewards(config);
 
   return {
-    config,
-    ...result
+    ...params,
+    result
   };
-}
+};
 
-const config = Config();
-
-App(config)
-  .then(x => console.log(JSON.stringify(x, null, 2)))
+makeUnixPipe(App)
```

Comment:
this got converted to take its config from standard in. it expects a json object which is not documented yet.